### ISSUE TITLE
Webhook parsing losing information due to encoding 

### ISSUE THREAD

**Issue by ThomasDoyle-byte** [Jericho/StrongGrid#347]
ParseInboundEmailWebhookAsync incorrectly sets the encoding for "html" and "text" to us-ascii, it should have detected the content encoding was utf-8.

Example email content:
This is a test email with ññ ñññ ññ basic HTML formatting

After parsing comes through as:
This is a test email with ?? ??? ?? basic HTML formatting

**Comment by Jericho** [comment]
Can you provide the payload posted to you by SendGrid? Part of the payload contains the encoding which was used by SendGrid to encode the content and StrongGrid attempts to respect this encoding. However, if the encoding is unknown, invalid, not supported on your platform, etc. we revert back to UTF-8 which will lead to some characters not being decoded properly but it's preferable than throwing an exception (which is what used to occur before we added the 'fallback' logic).

Anyway, long story short: my guess is that your content was encoded using an encoding that is not supported on your platform and StrongGrid used UTF-8 as the fallback encoding. Again, this is just a guess on my part and there's no way for me to know for sure unless you provide the payload.

**Comment by Jericho** [comment]
@BirgitteLoweJ experienced a similar issue and she inspired me to add the "fallback" logic. Maybe she can chime in and share some of her experiences dealing with content that was encoded with an encoding that StrongGrid can't handle.

**Comment by BirgitteLoweJ** [comment]
We had problems with the encoding, but we only needed the attachments, and havn't really cared about the content in the email. The fallback logic solved our issue.

**Comment by Jericho** [comment]
@ThomasDoyle-byte can you provide the requested additional info?

**Comment by Jericho** [comment]
I'm afraid I can't help you any further without additional information so I am closing this issue but I'll be happy to reopen if you provide the requested information.
